pkgs/desktops/gnome/extensions/default.nix

```diff
@@ -71,6 +71,7 @@ in rec {
       nohotcorner = throw "gnomeExtensions.nohotcorner removed since 2019-10-09: Since 3.34, it is a part of GNOME Shell configurable through GNOME Tweaks.";
       mediaplayer = throw "gnomeExtensions.mediaplayer deprecated since 2019-09-23: retired upstream https://github.com/JasonLG1979/gnome-shell-extensions-mediaplayer/blob/master/README.md";
       remove-dropdown-arrows = throw "gnomeExtensions.remove-dropdown-arrows removed since 2021-05-25: The extensions has not seen an update sine GNOME 3.34. Furthermore, the functionality it provides is obsolete as of GNOME 40.";
+      topicons-plus = throw "gnomeExtensions.topicons-plus removed since 2021-25-29: Upstream has abandoned it";
```

Comment:
There is a PR to enable GNOME 40 support (https://github.com/NixOS/nixpkgs/pull/123392)

Reply:
Upstream says they don't maintain the project actively anymore. There won't ever be wayland support and they suggest switching to https://github.com/ubuntu/gnome-shell-extension-appindicator instead. But if you are willing to maintain it nevertheless with the patches on top, I can of course keep it in.

Reply:
I will continue maintaining it. Unfortunately, there are still applications that still use the legacy X11 tray and won't work with appindicator.